lib/maxmind/geoip2/client.rb

```diff
@@ -245,11 +245,15 @@ def response_for(endpoint, model_class, ip_address)
       end
 
       def make_http_client
+        client_version = Gem::Specification.load('maxmind-geoip2.gemspec').version.version
+        httplib_version = Gem.loaded_specs['http'].version
```

Comment:
Similarly it appears we could use the `VERSION` constant from the `HTTP` class to know its version.

(Separately, I think we'd want to `require` the `rubygems` library before using these `Gem*` classes. I assume something else is loading them right now. I think we'll not need them with these changes though).